tests/integration/charts/cluster_with_stargate.yaml

```diff
@@ -15,6 +15,8 @@ stargate:
   ingress:
     host: stargate.127.0.0.1.nip.io
     enabled: true
+    cassandra:
```

Comment:
What's this extra block for here?

Reply:
that's what triggered the issue. When we enable cassandra ingress on stargate, we create an IngressRouteTCP for stargate with the infamous `kind` field which was removed from later versions of the Traefik CRDs.

Reply:
and so far we weren't catching it because only the full stack test has this, but it ain't running in CI.